cluster/gce/config-default.sh

```diff
@@ -25,7 +25,7 @@ MASTER_DISK_TYPE=pd-ssd
 MASTER_DISK_SIZE=${MASTER_DISK_SIZE:-20GB}
 MINION_DISK_TYPE=pd-standard
 MINION_DISK_SIZE=${MINION_DISK_SIZE:-100GB}
-REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-false}
+REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-true}
```

Comment:
Since the master node is unschedulable, it should be safe to flip the default value in config-test.sh as well. And then our e2e suite will have the same default behavior as the cluster creation scripts.